src/faebryk/libs/sexp/dataclass_sexp.py

```
@@ -311,13 +322,18 @@ def _decode[T](
                 f" {sp.assert_value} but is {value_dict[f.name]}"
             )
 
+    # Unprocessed values should be None is there aren't any,
+    # so they don't get reproduced etc...
+    if catch_all_fields and not unprocessed:
```

Convert an empty catch-all to `None`, so it doesn't show up in outputs